### PR TITLE
[SPARK-19869][SQL]move table related ddl from ddl.scala to tables.scala

### PR DESCRIPTION
**Pull request by windpiger**
## What changes were proposed in this pull request?

move table related ddl from ddl.scala to tables.scala

## How was this patch tested?
N/A

### CODE REVIEW

**Comment by windpiger** [comment]
cc @cloud-fan

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala [hunk 1]
```diff
@@ -167,599 +153,6 @@ case class DescribeDatabaseCommand(
   }
 }
```
Comment (gatorsmile) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala]: The remaining commands are database only. We can move the database related commands to there.

Comment (windpiger) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala]: ok~ thanks, another problem is now  the lines of tables.scala is 1594, this is too large, should we need to split it to reduce the  lines of a file?

**Comment by gatorsmile** [comment]
This change might cause people hard to track the change history.  Maybe we can close this PR?

**Comment by gatorsmile** [comment]
We are closing it due to inactivity. please do reopen if you want to push it forward. Thanks!

**Pull request closed without merging**